app/src/main/java/com/orange/ouds/app/ui/components/radiobutton/RadioButtonItemDemoScreen.kt

```diff
@@ -0,0 +1,131 @@
+import com.orange.ouds.app.ui.components.controlitem.ControlItemHelperTextCustomization
+import com.orange.ouds.app.ui.components.controlitem.ControlItemIconCustomization
+import com.orange.ouds.app.ui.components.controlitem.ControlItemInvertedCustomization
+import com.orange.ouds.app.ui.components.controlitem.ControlItemReadOnlyCustomization
+import com.orange.ouds.app.ui.components.controlitem.ControlItemTextCustomization
+import com.orange.ouds.app.ui.components.controlitem.addControlItemAttributes
+import com.orange.ouds.app.ui.components.onClickArgument
+import com.orange.ouds.app.ui.utilities.composable.CodeSnippet
+import com.orange.ouds.app.ui.utilities.composable.CustomizationBottomSheetScaffold
+import com.orange.ouds.app.ui.utilities.composable.CustomizationSwitchListItem
+import com.orange.ouds.app.ui.utilities.composable.CustomizationTextField
+import com.orange.ouds.app.ui.utilities.composable.DemoScreen
+import com.orange.ouds.app.ui.utilities.composable.LightDarkDemo
+import com.orange.ouds.core.component.OudsControlItem
+import com.orange.ouds.core.component.OudsRadioButtonItem
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.UiModePreviews
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun RadioButtonItemDemoScreen() = DemoScreen(rememberRadioButtonItemDemoState()) {
+    CustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+            ControlItemIconCustomization()
+            ControlItemDividerCustomization()
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_radioButton_radioButtonItem_outlined_label),
+                checked = outlined,
+                onCheckedChange = { outlined = it },
+            )
+            ControlItemInvertedCustomization()
+            ControlItemEnabledCustomization()
+            ControlItemReadOnlyCustomization()
+            ControlItemErrorCustomization()
+            ControlItemTextCustomization()
+            CustomizationTextField(
+                label = stringResource(R.string.app_components_radioButton_radioButtonItem_additionalText_label),
+                value = additionalText.orEmpty(),
+                onValueChange = { value -> additionalText = value }
+            )
+            ControlItemHelperTextCustomization()
+        }
+    ) {
+        LightDarkDemo {
+            RadioButtonItemDemo(state = this@DemoScreen)
+        }
+
+        RadioButtonItemDemoCodeSnippet(
+            state = this@DemoScreen,
+            modifier = Modifier
+                .padding(all = OudsTheme.spaces.fixed.medium)
+                .padding(top = OudsTheme.spaces.fixed.medium)
+        )
+    }
+}
+
+@Composable
+private fun RadioButtonItemDemo(state: RadioButtonItemDemoState) {
```

Comment:
This is not related to this PR, but when comparing this code with `CheckboxItemDemoScreen` I noticed that the `onCheckedChange` parameter in `CheckboxItemDemo` and the `onClick` parameter in `IndeterminateCheckboxItemDemo` are useless and could be removed because everything can be achieved using the checkbox item `state`.

Reply:
I will fix it in the "2 checkboxes" PR.